index.html

```diff
@@ -173,6 +172,42 @@ <h4>2018</h4>
       </ul>
     </section>
 
+    <!-- Modals -->
+
+    <section id="modal" class="modal">
+      <div class="modal__window">
+        <div class="modal__header">
+          <h2 class="modal__title">Tonic</h2>
+          <i class="modal__icon-close fas fa-times"></i>
+        </div>
+        <ul class="modal__list">
+          <li><h3 class="modal__company">Uber</h3></li>
+          <li class="circle"></li>
+          <li><h4 class="modal__role">Lead Developer</h4></li>
+          <li class="circle"></li>
+          <li><h4 class="modal__year">2018</h4></li>
+        </ul>
+        <!-- <div class="modal__img"></div> -->
+        <img class="modal__img" src="./media/snapshoot-portfolio-3.png" alt="Work preview for project Uber Navigation">
+        <div class="modal__info">
+          <p class="modal__desc">Lorem ipsum dolor sit amet 
+            consectetur adipisicing elit. Veniam architecto sint, 
+            quae sit nostrum accusamus magnam blanditiis iusto in 
+            ducimus eligendi voluptatibus commodi.
+
+          </p>
+          <div class="modal__tags-buttons-container">
+            <ul class="modal__tech"></ul>
+            <div class="modal__line"></div>
+            <div class="modal__buttons-container">
+              <a href="#" class="modal__button modal__button--see-live" target="_blank">See live<i class="modal__icon-button fas fa-external-link-alt"></i></a>
+              <a href="#" class="modal__button modal__button--see-src" target="_blank">See source<i class="modal__icon-button fab fa-github"></i></a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
```

Comment:
* This section is to be created dynamically according to the project requirement eg: [ You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.]  so you can wrap the functions or DOM elements creation with Window.onLoad() function.

Reply:
Hello @Davidosky007

I'm sorry, I left a lot of leftover code in my Modals Section. I fixed this already in the last commit... But what I'm not sure of is the following:

> all of the HTML in that section is created when the page loads.

What I get from that is that my HTML for the modals section should load normally when the page loads and then I need to fill it with JS using the information stored in my JS object. Wich is the current behavior on my page.

Can you please check my last commit and check if what I'm doing is correct?

Also, we haven't go through any function of the Window object yet.

Reply:
you need a window.onload() function to carry out this [ all of the HTML in that section is created when the page loads. ]